ambry-clustermap/src/main/java/com/github/ambry/clustermap/RecoveryTestClusterManager.java

```diff
@@ -59,6 +59,7 @@ public PartitionId getPartitionIdFromStream(InputStream stream) throws IOExcepti
     } catch (IOException e) {
       // for composite blob, it has the list of Blob IDs. It may not exist in the static cluster map.
       // swallow the exception and try with helixClusterManager
+      logger.trace("RecoveryTestClusterManager getPartitionIdFromStream exception", e);
```

Comment:
logger.error ?

Reply:
logger.error instead of trace ?

Reply:
Yes, I'm thinking about the level. 
Actually it hits very frequently, basically for all the composite blob, we hit for every item in the meta blob. The log will be overwhelmed by it.
That's why in original PR, if partitionIDStatus == null, I log the message but only use trace.

    if (partitionIdStatic == null) {
      duplicatingInputStream.reset();
      partitionIdStatic = helixClusterManager.getPartitionIdFromStream(duplicatingInputStream);
      logger.trace("DR: partition doesn't exist in staticClusterManager, try helixClusterManager {}", partitionIdStatic);
    }

Reply:
Hi Sanketh,
Thanks for the suggestion.
1.  For the IOException block, I'll change accordingly.  
2. The reason that I prefer adding log in partitionIdStatic == null block is no matter staticClusterManager throws exception or returns null, it's the same level of error. Throwing exception in this case is not more severe than return null. Basically we couldn't find the Blob in the static map. 
3. And staticClusterManager.getPartitionIdFromStream can change its implementation to return null only instead of throwing exception. In this case, it's still the same level of error. Checking null or not is more generic and our code logic won't be dependent on the implementation of the static cluster map.

Reply:
Yeah I agree with points 2 and 3. I don't have any comments there. My comment was only for point 1.